### DISCUSSION TITLE
Verbindungsfehlschlag über Nacht DTU - > WR

### DISCUSSION THREAD

**Discussion by Nfo2000** [tbnobody/OpenDTU#1981] [Q&A]
Hallo zusammen ich habe seit kurzem ein Problem. Ich habe vor nem guten Jahr die DTU inbetriebgenommen. Es werden Daten vom HM-600 abgerufen. Zu dem Zeitpunkt hat alles funktioniert und habe Daten erhalten.
Irgendwann kam ein Update und seit dem bekomme ich am nächsten Tag keine Verbindung mehr. Hell genug ist es. An einem klassischen Stromzähler generiert die Anlage 400W. Erst wenn ich die DTU neustarte dann bekomme ich wieder Aktualdaten. Aktuelle Updates wurden auch schon nachinstalliert aber keine Änderung.

**Comment by MetaChuh** [comment]
@Nfo2000

> Irgendwann kam ein Update und seit dem bekomme ich am nächsten Tag keine Verbindung mehr.

die updates werden rein manuell eingespielt, es kann kein update plötzlich irgendwann vorhanden sein.
was passiert, wenn du (ebenfalls manuell) auf die version downgradest, bei der die kommunikation geklappt hat ?

anmerkung: wenn jemand für dich unwissentlich ein opendtu update eingespielt hatte, frage ihn, welche firmware version davor aktiv war.

greetings,
metachuh

**Reply by Nfo2000** [reply]
Nein das Update habe ich selber eingespielt. Ich hab das älteste Update drübergebügelt. Immer noch bestehend
